isyfact-standards-doc/src/docs/antora/modules/isy-security/pages/konzept/inhalt.adoc

```diff
@@ -226,6 +226,75 @@ OpenID Connect Abfragen zur Identität eines Benutzers, OpenID-Token oder Custom
+
+Alle daran beteiligten Komponenten unterscheiden sich in ihrem Aufbau, ihrer Konfiguration und ihrer Funktionalität, so dass jede genau die speziellen Kommunikationskanal-Eigenheiten - siehe dazu Kapitel <<aufruf-weboberflaeche-portal>>, <<aufruf-service-gateway-ohne-authentication-gateway>>, <<aufruf-service-gateway-mit-authentication-gateway>> und <<ausfuehrung-batch-u-task>> bzw. <<aufruf-von-nachbarsystemen>> - am besten bedienen kann.
+Zur Umsetzung und sicherheitsrelevanten Trennung dieser Kommunikationskanäle werden diese als eigenständige Mandanten betrachtet und können somit auch als mehrere Instanzen pro Kanal eingerichtet werden.
+Dadurch ist es möglich fachliche Unterschiede über den gleichen technischen Kanal-Typ bei der Authentifizierung zu machen.
+
+Das nachfolgende Kapitel  erläutert diese Mandanten spezifische Behandlung.
+
+[[ways-of-authentication-as-multi-tenant]]
+==== Authentifizierungswege als Mandanten
+Die bisher aufgezeigten Authentifizierungswege werden in Bezug auf `isy-security` als jeweilig unterschiedliche Mandanten behandelt bzw. konfiguriert.
+Aber auch pro Authentifizierungsweg ist es möglich unterschiedliche Mandanten sprich mehrere Instanzen des gleichen Typs einzurichten.
+
+Um nun für jeden Mandanten eine eindeutige Zuordnung bzw. eine Mandanten spezifische Gruppierung der Properties herzustellen, muss der Property-Key im Pfad den Namen des Mandanten beinhalten:
+
+`spring.security.oauth2.resourceserver.jwt.tenants.<mandant-id>`
+
+Der Platzhalter `<mandant-id>` wird in der konkreten Konfigurationsausprägung auf den Namen des Mandanten gesetzt.
+So wäre beispielsweise für die drei Authentifizierungswege aus der Nutzungsschicht folgende Property-Pfade vorstellbar:
+
+- `spring.security.oauth2.resourceserver.jwt.tenants.*portal*`
+- `spring.security.oauth2.resourceserver.jwt.tenants.*sgw*`
+- `spring.security.oauth2.resourceserver.jwt.tenants.*agw*`
+
+Sollten für einen Authentifizierungskanal mehrere Instanzen notwendig werden, dann müssen sich deren Namen natürlich auch unterscheiden, wie beispielsweise:
+
+- `spring.security.oauth2.resourceserver.jwt.tenants.*sgw-1*`
+- `spring.security.oauth2.resourceserver.jwt.tenants.*sgw-2*`
+
+
+===== Mandanten spezifische Konfiguration am Beispiel Portal
+So bedarf beispielsweise die Authentifizierung einer externen WebClient-Anwendung über das _Portal_-Gateway eine Konfiguration des IAM-Service, die berücksichtigt, dass der Aufruf des IAM-Service hier eine Unterscheidung zwischen _internem_ und _externem_ Aufruf benötigt. Wie und wo diese Aufruf-Unterscheidung zum Tragen kommt, zeigt nun das nachfolgende Bild die Kommunikationswege auf, die ein WebClient über ein Portal zur Authentifizierung und Autorisierung durchläuft.
+
+.Kommunikationswege einer WebClient-Authentifizierung über Portal
```

Comment:
Ich sehe immer noch nicht, wo und wie das Portal (bzw. der apache tomcat und das mod_auth_openidc-Modul) die obige spring.security Konfiguration nutzen. Meines Wissens nach verwenden sie eigene Konfigurationsdateien. 
Die angegebenen spring.security-Pfade werden verwendet in resource servern, also (Geschäfts-) Anwendungen, die geschützte Ressourcen bereitstellen. (Siehe auch: IsyFact Referenzimplementierung Security)